notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb

```diff
@@ -179,18 +174,21 @@
     "                opset_version=14  # onnx opset version for export\n",
     "            )\n",
     "        print('Text Encoder successfully converted to ONNX')\n",
-    "    ov_text_encoder = mo.convert_model(input_model=onnx_path, compress_to_fp16=True)  # convert model to OpenVINO model\n",
-    "    serialize(ov_text_encoder, str(ov_path))  # serialize converted model for next usage\n",
-    "    print('Text Encoder successfully converted to IR')\n",
     "    \n",
     "\n",
     "if not TEXT_ENCODER_OV_PATH.exists():\n",
-    "    convert_encoder(pipe, TEXT_ENCODER_ONNX_PATH, TEXT_ENCODER_OV_PATH)"
+    "    convert_encoder_onnx(pipe, TEXT_ENCODER_ONNX_PATH)\n",
+    "    !mo --input_model $TEXT_ENCODER_ONNX_PATH --compress_to_fp16\n",
+    "    print('Text Encoder successfully converted to IR')"
    ]
   },
   {
-   "cell_type": "markdown",
-   "metadata": {},
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "PscvJUXf_hpm"
+   },
+   "outputs": [],
    "source": [
     "### U-net\n",
```

Comment:
Thanks for the quick fix on the mo @eaidova.
Request to include this content in a markdown cell.

Reply:
thank you, updated